src/shared/wazuhdb_op.c

```diff
@@ -201,6 +201,20 @@ int wdbc_parse_result(char *result, char **payload) {
     return retval;
 }
 
+/**
+ * @brief Validate the component of the query.
+ * @param[in] component Component to be validated.
+ * @return Enum ComponentType. COMPONENT_INVALID if the component is not valid.
+ */
+ComponentType validate_component(const char *component) {
```

Comment:
All the functions in this file have the prefix `wdbc_`

Reply:
Fixed!